ci-operator/step-registry/ipi/conf/aws/disconnectedprivatecluster/ipi-conf-aws-disconnectedprivatecluster-ref.metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+	"path": "ipi/conf/aws/disconnectedprivatecluster/ipi-conf-aws-disconnectedprivatecluster-ref.yaml",
```

Comment:
How about "ipi/conf/aws/disconnected-private/ipi-conf-aws-disconnected-private-ref.yaml"?

Reply:
After go through the ref commands shell file, I think we need break them down into different ref yaml files:
1. ipi/conf/mirror
In the ref, involve ICSP injection, pull secret modification, ca bundle injection
2. cco mode
3. private cluster setting (I think we can re-use ipi-conf)

Reply:
final chains would be like: 

```
chain:
  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-provision
  steps:
    - ref: ipi-install-rbac
    - ref: openshift-cluster-bot-rbac
    - ref: ipi-conf
    - ref: aws-provision-vpc-disconnected
    - ref: aws-provision-bastionhost
    - ref: aws-provision-cco-manual-users
    - ref: mirror-images-by-oc-adm                   <- mirror images
    - ref: ipi-conf-mirror-images                    <- pull secret & CA & ICSP configuration
    - ref: ipi-conf-aws-custom-vpc                   <- Subnet & AZ configuration
    - ref: ipi-conf-aws
    - ref: ipi-install-monitoringpvc
    - ref: proxy-config-generate
    - ref: ipi-install-install-full-control
    - ref: ipi-install-times-collection
  env:
  - name: ADD_INGRESS_RECORDS_MANUALLY
    default: "yes"
    documentation: "Enable manually create apps dns record."
  - name: PUBLISH
    default: "Internal"
    documentation: "Cluster publish strategy."
  - name: CREDENTIALS_MODE
    default: "Manual"
    documentation: "CCO credentials mode."
  documentation: |-
    Create an IPI cluster on AWS for QE e2e tests.

```

Reply:
How about rename `ipi-conf-mirror-images` to `ipi-conf-mirror`

Reply:
And for the new step `ipi-conf-aws-custom-vpc`, I'm going to replace `ipi-conf-aws-privatecluster` with this new one .